### PR TITLE
Add parameter definitions to summarize.R

### PR DESCRIPTION
**Pull request by awasyn**
# Description

- https://github.com/JRaviLab/MolEvolvR/issues/2
- https://github.com/JRaviLab/MolEvolvR/issues/33
## What kind of change(s) are included?

- [x] Bug fix (fixes check).
- [x] Enhancement (documentation).

# Checklist

Please ensure that all boxes are checked before indicating that this pull request is ready for review.

- [x] I have read and followed the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines.
- [x] I have searched for existing content to ensure this is not a duplicate.
- [x] I have performed a self-review of these additions (including spelling, grammar, and related).
- [x] I have added comments to my code to help provide understanding.
- [x] I have added a test which covers the code changes found within this PR.
- [x] I have deleted all non-relevant text in this pull request template.
- [x] **Reviewer assignment**: Tag a relevant team member to review and approve the changes.

### CODE REVIEW

**Comment by awasyn** [comment]
@d33bs

File: R/summarize.R [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # suppressPackageStartupMessages(library(rlang))
 # conflicted::conflict_prefer("filter", "dplyr")
 
-#' Filter by Domains
+#' filterByDomains
```
Comment (the-mayer) [inline R/summarize.R:13]: for consistency, modified the titles of function documentation to the function call.

File: R/summarize.R [hunk 2]
```diff
@@ -237,17 +278,30 @@ filterByFrequency <- function(x, min.freq) {
...
 #########################
-#' Summarize by Lineage
+#' MolEvolvR Summary
+#' @name MolEvolvR_summary
+#' @description
+#' A collection of summary functions for the MolEvolvR package.
+#' 
+NULL
+
+#' summarizeByLineage
```
Comment (the-mayer) [inline R/summarize.R:288]: Created a MolEvolvR_summary Rd block

File: R/summarize.R [hunk 3]
```diff
@@ -237,17 +278,30 @@ filterByFrequency <- function(x, min.freq) {
...
+#' column. Use "all" to skip filtering and include all rows.
 #'
 #' @importFrom dplyr arrange filter group_by summarise
 #' @importFrom rlang sym
 #'
-#' @return Describe return, in detail
+#' @return A tibble summarizing the counts of occurrences of elements in 
+#' the `column`, grouped by the `by` column. The result includes the number 
+#' of occurrences (`count`) and is arranged in descending order of count.
+#' @rdname MolEvolvR_summary
```
Comment (the-mayer) [inline R/summarize.R:304]: Grouped summary functions under MolEvolvR_summary with @rdname

**Review by the-mayer — APPROVED** [review]
Thanks @awasyn ! Great work on adding descriptions to the parameters and returns for these functions! I made a few changes, mostly to the way the Rd files are grouped within summarize.R. I used @rdname, instead of @name and grouped the summary functions under a common heading MolEvolvR_summary -- credit to you for recognizing the similar functionality! 

Additionally, I made some minor stylistic adjustments, which we may wish to apply to other function documentation. Will tag in future issues.

Merging shortly!

**Pull request merged**